x-pack/platform/test/functional/page_objects/copy_saved_objects_to_space_page.ts

```diff
@@ -40,20 +40,23 @@ export function CopySavedObjectsToSpacePageProvider({
       if (createNewCopies && overwrite) {
         throw new Error('createNewCopies and overwrite options cannot be used together');
       }
+
       if (!createNewCopies) {
         const form = await testSubjects.find('copy-to-space-form');
         // a radio button consists of a div tag that contains an input, a div, and a label
         // we can't click the input directly, need to click the label
         const label = await form.findByCssSelector('label[for="createNewCopiesDisabled"]');
         await label.click();
+
+        if (!overwrite) {
```

Comment:
This change is needed since the original test logic was executing clicks on disabled overwrite inputs.

Historically, this was a no-op, and executing the click had no adverse effects. With the recent change to `EuiCheckableCard` when there are no child interactive elements, clicking on the card results in selection of its radio button. A `disabled` input is considered non-interactive, which is the reason why this started failing.